webrender/src/device.rs

```diff
@@ -98,56 +99,56 @@ pub trait FileWatcherHandler : Send {
 }
 
 impl VertexFormat {
-    fn bind(&self, main: VBOId, instance: VBOId, offset: gl::GLuint, instance_stride: gl::GLint) {
-        main.bind();
+    fn bind(&self, gl_: &Rc<gl::Gl>, main: VBOId, instance: VBOId, offset: gl::GLuint, instance_stride: gl::GLint) {
```

Comment:
When I created the patch, I used self.gl() to get &gl::Gl. In this case, it caused the following error.
> error[E0502]: cannot borrow `*self` as immutable because `self.programs` is also borrowed as mutable

I tried again to remove &Rc<gl::Gl> and succeeded to do it. If &*self.gl is used instead of self.gl(), the compile error was addressed. In a next patch, I will use &*self.gl in places that caused the compile error.